File: Ninja Slayground 2.0/14. Sort An Array of 0s, 1s and 2s/Solution.java
// Learn the code, And copy and paste is not allowed. You are required to type...

// Follow this account, for all future code updates on GitHub.

import java.util.* ;
import java.io.*; 

public class Solution {
    public static void sortArray(ArrayList<Integer> arr, int n) {
        int low = 0, mid = 0, high = n - 1;
        
        while (mid <= high) {
            if (arr.get(mid) == 0) {
                Collections.swap(arr, low, mid); // Swapping elements using Collections.swap()
                low++;
                mid++;
            } else if (arr.get(mid) == 1) {
                mid++;
            } else {
                Collections.swap(arr, mid, high); // Swapping elements using Collections.swap()
                high--;
            }
        }
    }

}

File: Ninja Slayground 2.0/14. Sort An Array of 0s, 1s and 2s/Solution.py
#Learn the code, Copy and Paste is not allowed. You are required to type...

#Follow this account, for all future code updates on GitHub.

def sortArray(arr, n):
    low, mid, high = 0, 0, n - 1
    
    while mid <= high:
        if arr[mid] == 0:
            arr[low], arr[mid] = arr[mid], arr[low]
            low += 1
            mid += 1
        elif arr[mid] == 1:
            mid += 1
        else:
            arr[mid], arr[high] = arr[high], arr[mid]
            high -= 1
    return arr